src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/prison/PrisonEventListenerIntTest.kt

```diff
@@ -323,6 +324,14 @@ class PrisonEventListenerIntTest : MessagingMultiNodeTestBase() {
 
     val additionalInformation = AdditionalInformation(prisonNumber = prisonNumber)
     val domainEvent = DomainEvent(eventType = PRISONER_CREATED, personReference = null, additionalInformation = additionalInformation)
+    stubMatchScore(
+      MatchResponse(
+        matchProbabilities = mutableMapOf(
+          "0" to 0.9999999,
+          "1" to 0.9999991,
+        ),
+      ),
+    )
```

Comment:
sometimes, but not always, match is called here.
https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-person-record/3770/workflows/f72191dc-53e9-461b-8353-c2218776e6f3/jobs/11575/tests

Reply:
Is this a good idea? Might it be better to investigate why match is called at all?

Reply:
It should try match on create right (if theres candidates)? might be matching on the other records on something in this: `probationDomainEventAndResponseSetup` where similar name etc.